build/tools/make-winpe-i.MX8.cmd

```diff
@@ -0,0 +1,234 @@
+::
+:: Prepares a WinPE image for boot on i.MX6 or i.MX7
+::
+@echo off
+setlocal enableextensions disabledelayedexpansion
+
+set WINPE_DIR=%ProgramFiles(x86)%\Windows Kits\10\Assessment and Deployment Kit\Windows Preinstallation Environment\arm64
+set DEST=winpe_imx
+set FFU_DISK_NUM=0
+set WINPE_DRIVE_LETTER=D
+set SCRIPT_DIR=%~dp0
+
+:: make WIM mount directory
+rmdir /s /q mount > NUL 2>&1
+mkdir mount
+pushd mount
+set MOUNT_DIR=%CD%
+popd
+
+:: make directory to hold packages
+rmdir /s /q packages > NUL 2>&1
+mkdir packages
+pushd packages
+set PACKAGES_DIR=%CD%
+popd
+
+:: Parse options

```

Comment:
The options should be parsed before any work is done.  Before this point the script deletes the mount and packages directories in the local directory.  If I run make-winpe-i.mx8.cmd /?  it should just print out the help text, and not do anything else.